### PR TITLE
Better SWIG integration: autogen func & attributes

### PR DESCRIPTION
**Pull request by slurdge**

### CODE REVIEW

File: client/src/pm3.py [hunk 1]
```diff
@@ -1,98 +1,124 @@
 # This file was automatically generated by SWIG (http://www.swig.org).
-# Version 4.0.1
+# Version 3.0.12
```
Comment (iceman1001) [inline client/src/pm3.py:2]: How about you use the latest SWIG version?

**Comment by iceman1001** [comment]
I guess you talked with @doegox  about it,
He has this branch setup for this:
https://github.com/RfidResearchGroup/proxmark3/tree/libpm3_experiments

**Comment by slurdge** [comment]
Yes this pull request is for him to check out, it should be toward this branch & not master

**Comment by iceman1001** [comment]
haha,  sorry,  I missed you targeted his branch.

**Comment by doegox** [comment]
merged and rebased manually, thanks !

**Pull request closed without merging**